packages/did/src/Did.utils.ts

```diff
@@ -43,8 +50,10 @@ const FULL_KILT_DID_REGEX =
 const LIGHT_KILT_DID_REGEX =
   /^did:kilt:light:(?<auth_key_type>[0-9]{2})(?<identifier>4[1-9a-km-zA-HJ-NP-Z]{47,48})(?<encoded_details>:.+?)?(?<fragment>#[^#\n]+)?$/
 
-export const defaultDidKeySelection: DidKeySelectionHandler = (keys) =>
-  Promise.resolve(keys[0] || null)
+export const defaultVerificationDidKeySelection: DidVerificationKeySelectionHandler =
+  (keys) => Promise.resolve(keys[0] || null)
+export const defaultEncryptionDidKeySelection: DidEncryptionKeySelectionHandler =
```

Comment:
C: why do we need two functions, which do the same?

Reply:
Good point -> https://github.com/KILTprotocol/sdk-js/pull/473/commits/e7036eabe3b8f1c4c3d8e7497c6b3dd8b1c906fd